R/dates_secuTrial.R

```diff
@@ -97,33 +58,29 @@ dates_secuTrial.data.frame <- function(data, datevars, timevars, dateformat, dat
       newdatecol <- dates_secuTrial(data[, x], dateformat)
       # check for conversion of all else warn
       if (length(which(is.na(newdatecol))) > length(which(is.na(data[, x])))) {
-        warning(paste0("Not all dates were converted for\n",
+        if (warn) warning(paste0("Not all dates were converted for\n",
```

Comment:
As we just discussed I would prefer this not being disableable.Same goes for line 76.

Reply:
I would agree with that...if the user wants to silence warnings they can do so using the normal R method (i.e. via options)